exchanges/stream/websocket.go

```diff
@@ -568,6 +568,8 @@ func (w *Websocket) trafficMonitor() {
 						w.trafficTimeout)
 				}
 				trafficTimer.Stop()
+				w.Wg.Done() // without this the w.Shutdown() call below will deadlock
+				w.setTrafficMonitorRunning(false)
```

Comment:
Sorry, can you please swap these around so they are in the same call queue as before?

Reply:
Done